### PR TITLE
TRV-355: node add transfer function

### PR DESCRIPTION
**Pull request by cchen408**
# Description

* Add transfer functionality

### CODE REVIEW

File: traveler/examples/node/package.json [hunk 1]
```diff
@@ -2,14 +2,16 @@
   "name": "ct-traveler-api-examples",
   "version": "0.1.0",
```
Comment (bransorem) [inline traveler/examples/node/package.json:3]: Should we push this to `0.1.1`?

Comment (cchen408) [reply traveler/examples/node/package.json:3]: good idea

**Pull request merged**